src/cargo/ops/cargo_read_manifest.rs

```diff
@@ -121,7 +121,16 @@ fn read_nested_packages(path: &Path,
 
     let manifest_path = find_project_manifest_exact(path, "Cargo.toml")?;
 
-    let (manifest, nested) = read_manifest(&manifest_path, source_id, config)?;
+    let result = read_manifest(&manifest_path, source_id, config);
+
+    // Ignore malformed manifests
```

Comment:
Could you expand this comment as well with some rationale as to why malformed manifests are ignored?